lib/pengine/pe_output.c

```diff
@@ -450,6 +458,28 @@ cluster_summary(pcmk__output_t *out, va_list args) {
                      scheduler->localhost, last_written, user, client, origin);
     }
 
+    // Use the existing scheduler, but avoid scheduler output
+    pcmk__output_new(&verify_out, "none", NULL, NULL);
+    //scheduler = pe_new_working_set();
+    priv_orig = scheduler->priv;
+    scheduler->priv = verify_out;
+
+    verify_rc = pcmk__verify(scheduler, verify_out, scheduler->input);
+    scheduler->priv = priv_orig;
+    //pe_free_working_set(scheduler);
+    pcmk__output_free(verify_out);
+
+    if (verify_rc == pcmk_rc_ok) {
+        if (pcmk_is_set(section_opts, pcmk_section_verify)) {
+            PCMK__OUTPUT_LIST_HEADER(out, false, rc, "Cluster Summary");
+            out->list_item(out, NULL, "CIB syntax is valid");
+        }
+    } else {
+        /* If there are verification errors, always print a statement about that, even if not requested */
+        PCMK__OUTPUT_LIST_HEADER(out, false, rc, "Cluster Summary");
+        out->list_item(out, NULL, "CIB syntax has errors (for details, run crm_verify -LV)");
```

Comment:
looking at this again, let's omit the crm_verify arguments -- the user could be running against a CIB_file. it's up to the user to figure out what arguments they need.